.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       # Validate wrapper
       - name: Gradle Wrapper Validation
-        uses: gradle/wrapper-validation-action@v3.3.0
+        uses: gradle/wrapper-validation-action@v3.3.1
```

Comment:
See release notes in https://github.com/bufbuild/intellij-buf/pull/194 - the action has changed location.

Reply:
should this be `uses: gradle/actions/wrapper-validation@v3.3.1` then?